core/css.js

```diff
@@ -57,502 +55,503 @@ const inject = function(hasCss, pathToMedia) {
     return;
   }
   injected = true;
-  let text = CONTENT.join('\n');
-  CONTENT.length = 0;  // Garbage collect CSS content.
   if (!hasCss) {
     return;
   }
   // Strip off any trailing slash (either Unix or Windows).
   const mediaPath = pathToMedia.replace(/[\\/]$/, '');
-  text = text.replace(/<<<PATH>>>/g, mediaPath);
+  let cssContent = CONTENT.replace(/<<<PATH>>>/g, mediaPath);
+  // Cleanup the collected css content after injecting it to the DOM.
+  CONTENT = '';
```

Comment:
Do we need this step? (not sure!)

Reply:
I'm double checking with someone, but in the mean time I think it is fine to leave it.

Reply:
Yup, that cleanup is great!